File: exercicios/Python/1118/1118.py
i = 0
med = 0
novo = 0
new = True
while new == True:
    while i < 2:
        nota = float(input())
        if nota >= 0 and nota <= 10:
            i = i + 1
            med = med + nota
        if nota < 0 or nota > 10:
            print('nota invalida')
    med = med / 2
    print('media = {:.2f}'.format(med))

    if i == 2:
        i = 0
        med = 0
        
    while True:
            print("novo calculo (1-sim 2-nao)")
            novo = int(input())
            if novo == 2:
                new = False
                break
            elif novo == 1:
                new = True
                break

File: exercicios/R/1118/1118.r
input = file('stdin','r')
new = TRUE
while(new == TRUE){
    cont = as.integer(0)
    nota = as.double(0)
    op = as.integer(0)
    while(cont <2 ){
        n = as.double(readLines(input,n=1))

        if(n < 0.0 || n > 10.0){
            write("nota invalida",'')
        }else if(n >=0 && n <=10){
            cont = cont + 1
            nota  = nota + n
        }
    }
    write(sprintf("media = %.2f",nota/2),'')
    if(cont == 2){
        cont = 0
        nota = 0
        n = 0
    }
    while(TRUE){
        write("novo calculo (1-sim 2-nao)",'')
        op = as.integer(readLines(input,n=1))
        cont = 0
        if(op==2){
            new = FALSE
            break
        }else if(op == 1){
            new = TRUE
            cont = 0
            nota =0
            n = 0
            break
        }
    }
}